assests/javascript/index.js

```diff
@@ -6,10 +6,13 @@ const addTaskBox = document.getElementById("add-task-box");
 const addTaskInputErrorMassage = document.getElementById("title-error");
 const networkErrorMassage = document.getElementById("network-error");
 const taskList = document.getElementById("task-list");
+const headers = { "Content-Type": "application/json" };
```

Comment:
move this to the top or bottom of the imports and put a new line in between.

Reply:
ok